web_src/js/bootstrap.ts

```diff
@@ -90,7 +80,8 @@ function initGlobalErrorHandler() {
   }
   // then, change _globalHandlerErrors to an object with push method, to process further error
   // events directly
-  window._globalHandlerErrors = {_inited: true, push: (e) => processWindowErrorEvent(e)};
+  // @ts-expect-error -- this should be refactored to not use a fake array
+  window._globalHandlerErrors = {_inited: true, push: (e: ErrorEvent & PromiseRejectionEvent) => processWindowErrorEvent(e)};
```

Comment:
PS: I tried hard but failed to type this correctly because of the `number` return value that `push` requires. At least such hacks won't fly in the future anymore once type checking is enabled.